src/main/java/org/broadinstitute/hellbender/tools/funcotator/dataSources/gencode/GencodeFuncotation.java

```diff
@@ -39,107 +43,294 @@
+        this.hugoSymbolSerializedOverride = that.hugoSymbolSerializedOverride;
+        this.ncbiBuildSerializedOverride = that.ncbiBuildSerializedOverride;
+        this.chromosomeSerializedOverride = that.chromosomeSerializedOverride;
+        this.startSerializedOverride = that.startSerializedOverride;
+        this.endSerializedOverride = that.endSerializedOverride;
+        this.variantClassificationSerializedOverride = that.variantClassificationSerializedOverride;
+        this.secondaryVariantClassificationSerializedOverride = that.secondaryVariantClassificationSerializedOverride;
+        this.variantTypeSerializedOverride = that.variantTypeSerializedOverride;
+        this.refAlleleSerializedOverride = that.refAlleleSerializedOverride;
+        this.tumorSeqAllele1SerializedOverride = that.tumorSeqAllele1SerializedOverride;
+        this.tumorSeqAllele2SerializedOverride = that.tumorSeqAllele2SerializedOverride;
+        this.genomeChangeSerializedOverride = that.genomeChangeSerializedOverride;
+        this.annotationTranscriptSerializedOverride = that.annotationTranscriptSerializedOverride;
+        this.transcriptStrandSerializedOverride = that.transcriptStrandSerializedOverride;
+        this.transcriptExonSerializedOverride = that.transcriptExonSerializedOverride;
+        this.transcriptPosSerializedOverride = that.transcriptPosSerializedOverride;
+        this.cDnaChangeSerializedOverride = that.cDnaChangeSerializedOverride;
+        this.codonChangeSerializedOverride = that.codonChangeSerializedOverride;
+        this.proteinChangeSerializedOverride = that.proteinChangeSerializedOverride;
+        this.gcContentSerializedOverride = that.gcContentSerializedOverride;
+        this.referenceContextSerializedOverride = that.referenceContextSerializedOverride;
+        this.otherTranscriptsSerializedOverride = that.otherTranscriptsSerializedOverride;
+    }
+
+    //==================================================================================================================
+
     /**
      * @return A new builder for a {@link GencodeFuncotation}.
      */
     public static GencodeFuncotationBuilder getBuilder() {
         return new GencodeFuncotationBuilder();
     }
 
-    //==================================================================================================================
-
     /**
-     * @return An ordered {@link List} of {@link String} containing the field names that {@link GencodeFuncotation} produces.
+     * @return A {@link List} of {@link String} containing the field names that {@link GencodeFuncotation} produces.
      */
     public static List<String> getSerializedFieldNames() {
 
-        final List<String> fields = new ArrayList<>();
-
-        for(final Field f : GencodeFuncotation.class.getDeclaredFields() ) {
-            if ( !Modifier.isStatic(f.getModifiers()) ) {
-                fields.add( f.getName() );
-            }
-        }
-
-        return fields;
+        return Arrays.asList(
```

Comment:
I'll add in the version numbers.

Because this class is only for Gencode funcotations, I hard-coded the Gencode name in here.

Reply:
@LeeTL1220 However, it's worth noting that because of the way the overrides are working, you'll now need to specify the version in the field you want to override as well.